MMR.Randomizer/Resources/REQ_CASUAL.txt

```diff
@@ -58595,6 +58956,700 @@
         ],
         [
           "Fierce Deity's Mask Anywhere"
+        ],
+        [
+          "Goron Bomb Jump",
+          "Take Damage",
+          "Any Bomb Bag"
+        ]
+      ],
+      "TimeNeeded": "None",
+      "TimeAvailable": "None",
+      "TimeSetup": "None",
+      "IsTrick": false
+    },
+    {
+      "Id": "Deku Playground Rupee Displacement",
+      "RequiredItems": [],
+      "ConditionalItems": [],
+      "TimeNeeded": "None",
+      "TimeAvailable": "None",
+      "TimeSetup": "None",
+      "IsTrick": true,
+      "TrickTooltip": "Make the Rupees in Deku Playground fall by repeatedly using transformation masks or the Ocarina.",
+      "TrickCategory": "Easy"
+    },
+    {
+      "Id": "Ocarina of Time or a Transformation Mask",
+      "RequiredItems": [],
+      "ConditionalItems": [
+        [
+          "ItemOcarina"
+        ],
+        [
+          "MaskDeku"
+        ],
+        [
+          "MaskGoron"
+        ],
+        [
+          "MaskZora"
+        ],
+        [
+          "Fierce Deity's Mask Anywhere"
+        ]
+      ],
+      "TimeNeeded": "None",
+      "TimeAvailable": "None",
+      "TimeSetup": "None",
+      "IsTrick": false
+    },
+    {
+      "Id": "Keg Trial Access",
+      "RequiredItems": [],
+      "ConditionalItems": [
+        [
+          "AreaSnowheadTempleClear"
+        ],
+        [
+          "Use Fire Arrow"
+        ],
+        [
+          "Powder Keg Challenge with Hot Spring Water",
+          "Fierce Deity's Mask Anywhere"
+        ]
+      ],
+      "TimeNeeded": "None",
+      "TimeAvailable": "None",
+      "TimeSetup": "None",
+      "IsTrick": false
+    },
+    {
+      "Id": "SHT Pound Switch Puzzle",
+      "RequiredItems": [],
+      "ConditionalItems": [
+        [
+          "MaskGoron"
+        ],
+        [
+          "ItemHookshot"
+        ],
+        [
+          "MaskZora",
+          "Gainer"
```

Comment:
Zora can just jump over, no need to gainer. Or does that count as SHT Zora Jumps?

Reply:
Didn't know that Zora could simply jump over. SHT Zora Jumps can be added as a conditional for SHT Pound Switch Puzzle.